Hw3.py

```diff
@@ -0,0 +1,42 @@
+# Home task_1
+test_word = str(input('Enter any word:'))
+remove_letter = str(input('Specify some word\'s letter to be removed:'))
+corrected_sentence = (test_word.replace(remove_letter,''))
+print(f'Check what is done: {corrected_sentence}')
+#Home task_2
+string_initial_state = str(input('Type some sentence:'))
+string_medium_state = string_initial_state.lower()
+string_ready_state = string_medium_state.title()
+print(f'Get the correct typed string: {string_ready_state}')
+#Home task_3
+initial_string = str(input('Type some sentence and wait for the magic:'))
+finished_string = (initial_string[::-1])
+print(f'Magic is done: {finished_string}')
+#Home task_4
+string_1 = str(input('Enter any sentence you want:'))
+string_2 = str(input('Enter one more sentence:'))
+is_the_same = (string_1 == string_2)
+print(f'These values are identical: {is_the_same}')
```

Comment:
we can use the comparison immediately as a result print(f'These values are identical: {string_1 == string_2}')